pkg/agent/monitortool/monitor.go

```diff
@@ -444,11 +455,17 @@ func (m *NodeLatencyMonitor) monitorLoop(stopCh <-chan struct{}) {
 		case <-tickerCh:
 			// Try to send pingAll signal
 			m.pingAll(ipv4Socket, ipv6Socket)
+			// Introduce a delay to allow for response processing
+			time.Sleep(time.Duration(m.reportInterval / 2))
```

Comment:
A sleep is blocking and is definitely not the right approach for a select-based loop like this one.

Reply:
thanks,removed the sleep block.